egs/babel/asr1/local/prepare_data.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# Copyright 2018 Johns Hopkins University (Matthew Wiesner)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+# This is not necessarily the top-level run.sh as it is in other directories.   see README.txt first.
+
+. ./conf/lang.conf
+. ./path.sh
+. ./cmd.sh
+
+FLP=true
```

Comment:
What is FLP and LLP?

Reply:
I made it a little more flexible than it needed to be maybe. FLP is (Full language pack), the 40, 60, or 80h training set provided. LLP is limited language pack and it is always 10h.

Reply:
OK. You can just put that information somewhere (README?)